packaging/debian-unstable/snapd.postrm

```diff
@@ -0,0 +1,87 @@
+#!/bin/sh
```

Comment:
I really wish we could inline the snap-mgmt script into various packaging scripts. Not a -1 in any way, just a cry for help.